### PR TITLE
Product card

### PR DESCRIPTION
**Pull request by eyeriss42**
Implemented single product card
#8

### CODE REVIEW

File: client/src/Components/ProductCard.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  creator: string;
+  purchaseHistory: number;
+};
+
+export default function ProductCard({
+  name,
+  description,
+  image,
+  creator,
+  purchaseHistory,
```
Comment (thelmaboamah) [inline client/src/Components/ProductCard.tsx:17]: an alternative so you could accept fewer props is to just accept one prop (product perhaps) and then access name, description, etc... on the product object that's passed in.

File: client/src/App.tsx [hunk 2]
```diff
@@ -1,33 +1,22 @@
-import { useState } from "react";
-import reactLogo from "./assets/react.svg";
-import viteLogo from "/vite.svg";
 import "./App.css";
```
Comment (thelmaboamah) [inline client/src/App.tsx:1]: might be good to remove these vite styles.

File: client/tsconfig.node.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
     "composite": true,
     "skipLibCheck": true,
     "module": "ESNext",
-    "moduleResolution": "bundler",
+    "moduleResolution": "Bundler",
```
Comment (thelmaboamah) [inline client/tsconfig.node.json:6]: i'm curious why this was necessary?

**Review by thelmaboamah — COMMENTED** [review]
left a few minor suggestions and had a question about the tsconfig change

File: client/src/App.tsx [hunk 4]
```diff
@@ -1,33 +1,22 @@
...
+import ProductCart from "./Components/ProductCard";
```
Comment (pearlescence-m) [inline client/src/App.tsx:2]: Perhaps, should have a folder name with lower case 'components' in line with other pull requests?

**Review by meru-a — CHANGES_REQUESTED** [review]
Great job with this pull request! Everything looks great and works as expected. However, I do have 2 small comments:

1. As we agreed maybe after this PR was created, the merging should go first from your branch to the `staging` branch.
2. If css styles are added to this component, I wonder if having one common way of displaying it across the page/component would be beneficial for future refactoring. Meaning, perhaps, we could have either CSS-in-JS or a separate .css file for styling, but this is completely up to you, authors :)

File: client/src/Components/ProductCard.tsx (hunk 1 above)
Comment (larkinds) [reply client/src/Components/ProductCard.tsx:17]: Ok, totally cool with this. Let's make sure it's our group norm :)

**Review by meru-a — APPROVED** [review]

**Pull request merged**